lib/src/list_notifier.dart

```diff
@@ -5,23 +5,24 @@ import 'package:flutter/foundation.dart' show ChangeNotifier, ValueListenable;
 import 'package:functional_listener/functional_listener.dart';
 
 /// A List that behaves like `ValueNotifier` if its data changes.
-/// It does not compare the elements on bulk operations
+/// It does not compare the elements on bulk operations like `addAll` or
+/// `replaceRange` but will notify listeners if the list changes.
 ///
-/// If you set [notificationMode] to [normal] `ListNotifier` will compare if a
-/// value passed is equal to the existing value.
-/// Like `list[5]=4` if the content at index 4 is equal to 4 and only call
-/// `notifyListeners` if they are not equal.
+/// If you want to compare the elements you can use the
+/// `CustomNotifierMode.normal` mode and provide a custom equality function.
 ///
-/// If you set [notificationMode] to [manual] you have to call `notifyListeners`
-/// manually to notify all listeners.
-/// Like `list.notifyListeners();` if you want to notify all listeners.
+/// If you want to prevent the list from notifying listeners you can use the
+/// `CustomNotifierMode.manual` mode.
 ///
-/// If you set [notificationMode] to [always] `ListNotifier` will always notify
-/// all listeners if the list changes.
-/// Like `list[5]=4` will always notify all listeners.
+/// If you want to notify listeners on every change you can use the
+/// `CustomNotifierMode.always` mode.
 ///
-/// To allow atomic changes `ListNotifier` supports a single level of
-/// transactions
+/// The functions that will always notify listeners unless [notificationMode]
```

Comment:
I would add `will always notifiy listeners without comparing the existing value`

Reply:
Done ✅

Reply:
@escamoteur, it's already there at lines `[35, 35]` 🙋🏼‍♂️ 

![screenshot](https://github.com/escamoteur/listenable_collections/assets/29831987/43eb5185-d50e-4c76-8bf0-dfd0faa65dd1)